core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -140,19 +140,19 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
     background-color: transparent;
     border: none;
     box-sizing: border-box;
-    color: #000000;
+    color: #000;
     font-size: 18px;
     font-weight: bold;
     min-height: 40px;
     min-width: 100px;
     text-align: center;
   }
   .oppia-pagination-button:hover {
-    border: 2px solid #000000;
+    border: 2px solid #000;
     border-radius: 5px;
   }
   .oppia-pagination-button[disabled] {
-    color: #0000003D;
+    color: rgba(0, 0, 0, 0.239);
```

Comment:
Can we use hex color code and have opacity mentioned separately? Ditto below

Reply:
Done

Reply:
Not done?

Reply:
Done